modules/darwinConfigurations.macos-01/arm64-setup-notes.md

```diff
@@ -23,3 +23,13 @@ This will provide compatibility with x86_64-darwin
 ```command
 softwareupdate --install-rosetta --agree-to-license
 ```
+
+(for VM support, see tart.run)
```

Comment:
is tart already used somwhere?

Reply:
not yet, I just added some hints how to install it.

Reply:
All runners are now running via tart